PyInstaller/depend/analysis.py

```diff
@@ -925,12 +926,17 @@ def get_bootstrap_modules():
                 mod_name = os.path.join('lib-dynload', mod_name)
             loader_mods.append((mod_name, mod_file, 'EXTENSION'))
     # NOTE:These modules should be kept simple without any complicated dependencies.
-    loader_mods +=[
+    loader_mods += [
         ('struct', os.path.abspath(mod_struct.__file__), 'PYMODULE'),
-        ('pyimod01_os_path', os.path.join(loaderpath, 'pyimod01_os_path.pyc'), 'PYMODULE'),
-        ('pyimod02_archive',  os.path.join(loaderpath, 'pyimod02_archive.pyc'), 'PYMODULE'),
-        ('pyimod03_importers',  os.path.join(loaderpath, 'pyimod03_importers.pyc'), 'PYMODULE'),
-        ('pyimod04_ctypes',  os.path.join(loaderpath, 'pyimod04_ctypes.pyc'), 'PYMODULE'),  # noqa: E501
-        ('pyiboot01_bootstrap', os.path.join(loaderpath, 'pyiboot01_bootstrap.py'), 'PYSOURCE'),
+        ('pyimod01_os_path', os.path.join(loaderpath,
+                                          'pyimod01_os_path.pyc'), 'PYMODULE'),
+        ('pyimod02_archive', os.path.join(loaderpath,
+                                          'pyimod02_archive.pyc'), 'PYMODULE'),
+        ('pyimod03_importers',
+         os.path.join(loaderpath, 'pyimod03_importers.pyc'), 'PYMODULE'),
+        ('pyimod04_ctypes', os.path.join(loaderpath, 'pyimod04_ctypes.pyc'),
+         'PYMODULE'),  # noqa: E501
+        ('pyiboot01_bootstrap',
+         os.path.join(loaderpath, 'pyiboot01_bootstrap.py'), 'PYSOURCE'),
```

Comment:
This block should probably be excluded from formatting, as it now looks messier than before. And we can remove the noqa if we will increase line-size limit.